contributing/code/core_team.rst

```diff
@@ -168,8 +168,28 @@ All code must be committed to the repository through pull requests, except for
 :ref:`minor change <core-team_minor-changes>` which can be committed directly
 to the repository.
 
-**Mergers** must always use the command-line ``gh`` tool provided by the
-**Project Leader** to merge the pull requests.
```

Comment:
I think we should keep this line, otherwise it could be confused with https://cli.github.com, WDYT?